courses/serializers_test.py

```diff
@@ -156,26 +156,34 @@ def test_serialize_course(mock_context, is_anonymous, all_runs):
     )
 
     page = CoursePageFactory.create(course=course)
+
+    # create products for all courses so the serializer shows them
+    for run in CourseRun.objects.all():
+        ProductVersionFactory.create(product=ProductFactory(content_object=run))
```

Comment:
I think this should be `ProductFactory,create`, there are a few other similar usages in the other tests as well.

Reply:
Ah, good old copy/paste. Will fix